umap/admin.py

```diff
@@ -77,7 +82,13 @@ class UserAdmin(CSVExportMixin, UserAdminBase):
         "last_name",
         "last_login",
         "date_joined",
+        "maps",
     ]
+    list_display = list(UserAdminBase.list_display) + ["maps"]
+
+    def maps(self, obj):
```

Comment:
`maps_count` would be more explicit (in case we want to return `ids` at some point)